### DISCUSSION TITLE
Watermarks for Panels

### DISCUSSION THREAD

**Discussion by PriyanshBordia** [grafana/grafana#43541] [Dashboards, Panels & Visualizations]
- Option to add Images that can be used as watermarks for panels.

**Comment by MartinOps666** [comment]
+1

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82965. Feel free to continue the discussion around this there. Thank you!
